x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -15,301 +15,276 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
   {
     key: 'linux.advanced.agent.connection_delay',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
```

Comment:
@jmiller263 we should set these up to all use `i18n.translate()`.

Reply:
oh yeah. the `key` fields as well or just the `documentation` fields?

Reply:
just the `documentation` fields since that will be doc text for people to read

Reply:
the `key` is too, no?